server/binding.js

```diff
@@ -436,19 +445,19 @@ binding.loadHistogram_ = function(file) {
     return null;
   }
 
-  var n = buf.length / 8;
+  var n = buf.length / 12;
```

Comment:
Maybe create a variable for 12, and explain in the variable comment that there are 1 int and 1 double on each line.